_stbt/transition.py

```diff
@@ -127,6 +127,7 @@ def press_and_wait(self, key):
         original_frame, _ = next(self.frames)
         self.dut.press(key)
         press_time = time.time()
+        stbt.debug("transition: %.3f: Pressed %s" % (press_time, key))
```

Comment:
Perhaps this log belongs in `DeviceUnderTest.press` if we want to be sure that we're logging every press.

Reply:
We can add that later (and have `stbt.press` return the time of the keypress for `press_and_wait` to use).